week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -5,42 +5,62 @@
 */
 
 var person1 = {
-    name: "Alice",
-    age: 25
+  name: "Alice",
+  age: 25
 };
 
 var person2 = {
-    name: "Bob",
-    age: 30
+  name: "Bob",
+  age: 30
 };
 
 var person3 = {
-    name: "John",
-    age: 20
+  name: "John",
+  age: 20
 };
 
 /* 
 DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
 
+var persons = [person1, person2, person3]; // Complete here
 
-var persons = // Complete here
-
-var personNames = // Complete here
-
-var personsYoungerThan28YearsOld = // Complete here
+var personNames = function getName(person) {
+  // Complete here
+  return person.name;
+};
+persons.map(getName);
 
+var personsYoungerThan28YearsOld = function youngerThan(person) {
```

Comment:
That's right too. When you're assigning a function to a variable it's often best not the give the function a name (think anonymous function) so you don't have two names.